test/Microsoft.AspNet.Mvc.Razor.Test/RazorPageTest.cs

```diff
@@ -311,7 +311,7 @@ public void IsSectionDefined_ThrowsIfPreviousSectionWritersIsNotRegistered()
             {
                 { "baz", _nullRenderAsyncDelegate }
             };
-            page.RenderBodyDelegateAsync = CreateAsyncWriteDelegate("body-content");
+            page.BodyContent = new HtmlEncodedString("body-content");
```

Comment:
Why not `HtmlString`? We don't use `HtmlEncodedString` anywhere in the MVC repo.